addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1924,6 +1931,9 @@ var SnippetsMenu = Widget.extend({
      *          (might be async when an editor must be created)
      */
     _activateSnippet: async function ($snippet, previewMode, ifInactiveOptions) {
+        if (this.options.enableTranslation) {
+            return;
+        }
```

Comment:
It's only needed because called when enabling an invisible element, right? Maybe add a comment saying that :+1: